### PR TITLE
Fix #31

### LINKED ISSUE #31: Years before 2004 not working
E.g.

``` r
stats19::dl_stats19(year = 1985)
#> Error in find_file_name(years = year, type = type): No files of that type exist
```

<sup>Created on 2018-12-05 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Hey @layik can you review this please?

Note - this is what I now get so I think it fixes the issue, using the more modular and years-focussed function `check_years()`:

```
dl_stats19(year = 1985)
Year not in range, changing to 1979
No files of that type found, showing all files for the year
File to download: Stats19-Data1979-2004.zip
This file is over 240 MB in size.
Once unzipped it is over 1.8 GB.
Attempt downloading from: 
   http://data.dft.gov.uk.s3.amazonaws.com/road-accidents-safety-data/Stats19-Data1979-2004.zip
```

File: R/utils.R [hunk 1]
```diff
@@ -37,7 +36,11 @@ check_year = function (year) {
     msg = paste0("Years must be in range 1979:", current_year() - 1)
     stop(msg)
   }
-  year
+  if(year %in% 1980:2003) {
```
Comment (layik) [inline R/utils.R]: Should we not make this 1979:2004?

**Comment by layik** [comment]
Looks good to me @Robinlovelace.

File: R/utils.R (hunk 1 above)
Comment (layik) [reply R/utils.R]: as per `msg = paste0("Years must be in range 1979:", current_year() - 1)`

Comment (Robinlovelace) [reply R/utils.R]: Yes but if its 1979 or 2004 it doesn't need to be changed: will already match the right filename

**Pull request merged**